code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -480,27 +515,30 @@
 	category = "Defensive"
 	cost = 1
 
+//THESE ARE NOT PURCHASABLE SPELLS! They're references to old spells that got removed + shit that sounds stupid but fun so we can painfully lock behind a dimmer component
+
+/datum/spellbook_entry/challenge
+	name = "Take the Challenge"
+	refundable = FALSE
+	category = "Challenges"
+	buy_word = "Accept"
+
+/datum/spellbook_entry/challenge/multiverse
+	name = "Multiverse Sword"
+	desc = "The Station gets a multiverse sword to stop you. Can you withstand the hordes of multiverse realities?"
+
+/datum/spellbook_entry/challenge/antiwizard
+	name = "Friendly Wizard Scum"
+	desc = "A \"Friendly\" Wizard will protect the station, and try to kill you. They get a spellbook much like you, but will use it for \"GOOD\"."
+
 /datum/spellbook_entry/summon
 	name = "Summon Stuff"
 	category = "Rituals"
 	refundable = FALSE
 	buy_word = "Cast"
```

Comment:
this might be a bit out of scope for this PR, but it'd be nice if the summon guns/magic/events spells could be made available even on dynamic if the only antags spawned by dynamic mode so far have been wizards (or if all of the dynamic-spawned antags in the round other than wizards are dead).